src/SphericalHarmonics.jl

```diff
@@ -146,6 +147,18 @@ function getindex(S::SphericalHarmonic{T}, x::ZSphericalCoordinate, K::BlockInde
     exp((lgamma(ℓ+m̃)+lgamma(ℓ-m̃)-2lgamma(ℓ))/2)*sqrt((2ℓ-1)/(4π)) * exp(im*m*x.φ) * sin(θ/2)^m̃ * cos(θ/2)^m̃ * Jacobi{real(T)}(m̃,m̃)[x.z, ℓ-m̃]
 end
 
+function getindex(S::RealSphericalHarmonic{T}, x::ZSphericalCoordinate, K::BlockIndex{1}) where T
+    ℓ = Int(block(K))
+    k = blockindex(K)
+    m = k-ℓ
```

Comment:
Here is where I would differ from the complex case and instead of `m >0` vs `m < 0` I would do `iseven(k)` vs `isodd(k)`.

@MikaelSlevinsky This differs from your convention. Do you strongly disagree? One argument in favour of Mikael's approach is that it is consistent with FFTW.

That said, I'm planning to make the axes in the complex case indexed from `1-k:k-1` so that we get true negative indexing in which case the index matches `m`. There's an argument for doing this also in the real case. But assigning negative `m` to sine terms just feels wrong to me.

Reply:
I don't have any strong feelings about any conventions. I chose the conventions that make the transforms carry the least amount of silly flops (even if they're just O(n<sup>2</sup>)), which implies a more maintainable codebase, personally. You can build on this however you feel comfortable!